docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -0,0 +1,211 @@
+=== Events and Functions
+
+Select any Events and Function calls that you wish to monitor. Selecting Events and Functions acts as an OR clause, in that if a transaction matches ANY of the selected Events or Functions, it will trigger notification.
+
+NOTE: An ABI is required to configure Events and Function detection
+
+NOTE: If no Events or Functions are specified, then the Sentinel will monitor ALL transactions to or from your monitored address including those that do not involve functions or events (e.g., sending ether to your contract's address).  If Events or Functions are selected, then ONLY those that are selected are monitored.
+
+IMPORTANT: At this time, Sentinels do not monitor internal function calls.
+
+[[specify-conditions]]
+== What are Conditions?
+
+Conditions act as filters that allow you to narrow the transactions even further.  These are entered as expressions and offer a great deal of flexibility.  Conditions are very much like Javascript expressions.
+
+NOTE: If you want to receive ALL transactions that involve your selected events/functions, then do not specify any conditions.
+
+General Syntax
+
+* Conditions can use *AND*, *OR*, *NOT* and *()*
+
+* Conditions can use *==*, *&lt;*, *&gt;*, *&gt;=*, *&lt;=* to compare
+
+* Number values can be referred to by Hex (0xabc123) or Decimal (10000000000)
+
+* String values can only be compared via *==*
+
+* Includes basic math operators: *+*, *-*, ***, */*, *^*
+
+=== Transaction Conditions
+
+IMPORTANT: If a transaction condition is specified, then a transaction MUST meet this condition in order to trigger a notification.
+
+NOTE: To accomodate comparisons for checksum and non-checksum addresses, expressions are case-insensitive.
+
+Transaction Conditions can refer to the following properties
+
+* *to* is the _to_ address for the transaction
+
+* *from* is the _from_ address for the transaction
+
+* *gasPrice* is the price of gas sent in the transaction
+
+* *gasUsed* is the amount of gas used in the transaction
+
+* *value* is the value sent in the transaction
+
+* *nonce* is the nonce for the specific transaction
+
+* *status* is a derived value and can be compared with *"success"* or *"failed"*
+
+==== Example Conditions
+
+Transactions that are reverted
+
+[source,jsx]
+----
+status == "failed"
+----
+
+Transactions excluding those from 0xd5180d374b6d1961ba24d0a4dbf26d696fda4cad
+
+[source,jsx]
+----
+from != "0xd5180d374b6d1961ba24d0a4dbf26d696fda4cad"
+----
+
+Transactions that have BOTH a gasPrice higher than 50 gwei AND a gasUsed higher than 20000
+
+[source,jsx]
+----
+gasPrice > 50000000000 and gasUsed > 20000
+----
+
+=== Event and Function Conditions
+
+Event and Function conditions further narrow the set of transactions that trigger notification.  These can refer to arguments in the signature either by name (if argument is named) or by index (e.g., $0, $1...).  The variables that are available to you are indicated in the user interface as you specify these functions.
```

Comment:
```suggestion
Event and Function conditions further narrow the set of transactions that trigger notification.  These can refer to arguments in the signature either by name (if the argument is named) or by index (e.g. $0, $1...).  The variables that are available to you are indicated in the user interface as you specify these functions.
```

Reply:
Had to look this up: ` In US English, "e.g." is usually followed by a comma. It is not followed by a comma in other English-speaking countries.`

Reply:
Interesting! Let's go with the US convention then.